test/built-ins/Object/create/15.2.3.5-0-2.js

```diff
@@ -4,12 +4,6 @@
 /*---
 es5id: 15.2.3.5-0-2
 description: Object.create must exist as a function taking 2 parameters
-includes: [runTestCase.js]
 ---*/
 
-function testcase() {
-  if (Object.create.length === 2) {
-    return true;
-  }
- }
-runTestCase(testcase);
+assert.sameValue(Object.create.length, 2, 'Object.create.length: ');
```

Comment:
It sounds like you would like the test report to include actual and expected values, even for satisfied assertions. If Test262 does ever implement a reporter, couldn't this information be emitted programatically? I ask because if so, that would let us avoid duplicating information when writing assertion messages.